cms/envs/common.py

```diff
@@ -1178,6 +1178,9 @@
     'openedx.features.course_duration_limits',
     'openedx.features.content_type_gating',
     'experiments',
+
+    # Reference grades after migration - may need to move it
+    'lms.djangoapps.grades.apps.GradesConfig',
```

Comment:
We should fix whatever problem was preventing studio from starting without grades.

Reply:
How can we determine/troubleshoot it?

Reply:
For future reference: the issue is that CMS imports something from openedx which imports something from LMS courseware which imports something from the grades app, which isn't (before this PR) installed in studio.